docs/assets/css/style.css

```diff
@@ -83,6 +83,15 @@ h5 {
  margin: 0.5rem;
  padding-top: 0.5rem;
 }
+#filter-group {
+ margin-left: 10px;
+}
 .filter-title {
  color: grey;
 }
+
+input[type='checkbox'] {
+ -webkit-appearance: radio; /* Safari, Chrome, Opera */
+ -moz-appearance: radio; /* Firefox */
+ -ms-appearance: radio; /* Not supported */
```

Comment:
They way I understand [1]:

1. In a checkbox group, a user can select more than one option.
2. A user can select only one option in a radio button group.

And looking on how we want to use this, I think checkbox is the way to go. So why are we making `checkbox` look like `radio`? .

[1]: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/

Reply:
@nerdlet ☝️

Reply:
@kilemensi True True. The way the designs are,  we require them to look like radio buttons but behave like checkboxes 

![image](https://user-images.githubusercontent.com/3693563/58476666-21770a00-815a-11e9-8813-cf9de105a760.png)

Reply:
If _that_ is what the design requires, then yeah go ahead @nerdlet. But please raise the issue when you demo it to team: Seeing `radio` buttons used like this is confusing.